fabric-ca-client/lib/FabricCAClient.js

```diff
@@ -70,21 +65,21 @@ const FabricCAClient = class {
 
 		this._cryptoPrimitives = cryptoPrimitives;
 
-		logger.debug('Successfully constructed Fabric CA client from options - %j', connect_opts);
+		logger.debug(`Successfully constructed Fabric CA client from options - ${JSON.stringify(connect_opts)}`);
```

Comment:
If somehow `connect_ops` has a circular reference, `JSON.stringify()` will throw an error. In the previous version the logging framework would be using the Node _util_ package, which handles circular references without an error.

Actually both `util.format()` with `%o` and `util.inspect()` do a better job in this case, but not throwing an error in logging is critical.

```
> const a = {};
> const b = {a};
> a.b = b;
> JSON.stringify(a);
Thrown:
TypeError: Converting circular structure to JSON
    at JSON.stringify (<anonymous>)
> util.format('%j', a);
'[Circular]'
> util.format('%o', a);
'{ b: { a: [Circular] } }'
> util.inspect(a);
'{ b: { a: [Circular] } }'
```

Reply:
Thanks for your clarify and reminds! I would use the code fashion combine with string literal and `util.inspect`